hushline/routes.py

```diff
@@ -72,28 +72,31 @@ def inbox() -> Response | str:
         # Redirect if not logged in
         if "user_id" not in session:
             flash("Please log in to access your inbox.")
-            return redirect(url_for("login"))
+            return redirect(url_for("login")), 302
 
         logged_in_user_id = session["user_id"]
         requested_username = request.args.get("username")
         logged_in_username = User.query.get(logged_in_user_id).primary_username
 
         if requested_username and requested_username != logged_in_username:
-            return redirect(url_for("inbox"))
+            return redirect(url_for("inbox")), 302
 
         primary_user = User.query.get(logged_in_user_id)
         messages = (
             Message.query.filter_by(user_id=primary_user.id).order_by(Message.id.desc()).all()
         )
         secondary_users_dict = {su.id: su for su in primary_user.secondary_usernames}
 
-        return render_template(
-            "inbox.html",
-            user=primary_user,
-            secondary_username=None,
-            messages=messages,
-            is_secondary=False,
-            secondary_usernames=secondary_users_dict,
+        return (
```

Comment:
200 is the default code, so this is unnecessary

Reply:
👍